src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/personalisation/appellant/letter/AppellantInternalCaseSubmitAppealOutOfTimeWithExemptionLetterPersonalisation.java

```diff
@@ -43,8 +43,10 @@ public String getTemplateId() {
 
     @Override
     public Set<String> getRecipientsList(final AsylumCase asylumCase) {
-        return Collections.singleton(getAppellantAddressAsList(asylumCase).stream()
-            .map(item -> item.replaceAll("\\s", "")).collect(Collectors.joining("_")));
+        return inCountryAppeal(asylumCase) ? Collections.singleton(getAppellantAddressAsList(asylumCase).stream()

```

Comment:
I think we are using this same structure for several notifications maybe is time to refator this into a utility method?

Reply:
Thanks I will add this and replace the personalisations we already have as well